scripts/gcloud_adapter.py

```diff
@@ -80,3 +186,21 @@ def deploy_application(app_yaml_path, app_name, version=None):
     if version is not None:
         args.append('--version=%s' % version)
     subprocess.check_output(args)
+
+
+def flush_memcache(app_name):
+    """Flushes memcache for the server.
+
+    Args:
+        app_name: str. The name of the GCloud project.
+    """
+    # remote_api_shell.py tries to read history file for the shell. It results
```

Comment:
I don't think I'm that keen on removing this file. It seems important to have a log.

How come the file can't be read, but can be deleted? That seems a bit weird.

In any case, if we can't find a clean way to do this, I think it is fine to prompt the user to flush memcache (and open up the page so it's easy for them to do).

Reply:
I am trying to figure out a fix for this and will make updates accordingly.

Reply:
Updated this ([Ref](https://cloud.google.com/appengine/docs/standard/python/tools/remoteapi#using_the_remote_api_in_a_local_client)). I just need to check the app_id for prod servers to ensure that this is correct.

Reply:
Checked and updated.